src/network/networkd-manager.c

```diff
@@ -713,15 +706,19 @@ int manager_enumerate(Manager *m) {
                 return log_error_errno(r, "Could not enumerate neighbors: %m");
 
         r = manager_enumerate_nexthop(m);
-        if (r < 0)
-                return log_error_errno(r, "Could not enumerate nexthop rules: %m");
+        if (r == -EOPNOTSUPP || (r == -EINVAL && mac_selinux_enforcing()))
```

Comment:
hmm what's the brackground of this weird selinux check? maybe could use a comment?

Reply:
It comes from 92b555aaabf710e0a672a7244e8c0e3963075133, Added a brief comment in the code.